core/math/quaternion.h

```diff
@@ -76,6 +76,8 @@ struct _NO_DISCARD_ Quaternion {
 
 	Vector3 get_axis() const;
 	real_t get_angle() const;
+	Quaternion get_twist(const Vector3 &v) const;
```

Comment:
```suggestion
	Quaternion get_twist(const Vector3 &p_axis) const;
```
Prefix arguments with `p_`, and make descriptive